### PR TITLE
v10-fixes

### PR DESCRIPTION
**Pull request by blagoev**
v10 fixes

### CODE REVIEW

**Comment by kraenhansen** [comment]
Is it a temporary fix to store a transpiled version of `realm-network-transport` in the repository? And .. why are we doing that now that it's available via NPM?

**Comment by blagoev** [comment]
`realm-network-transport.tgz` is either a leftover or a wrong merge.

**Comment by blagoev** [comment]
@kraenhansen This file is a left over and should have been removed in this PR
https://github.com/realm/realm-js/pull/291

File: types/index.d.ts [hunk 1]
```diff
@@ -322,17 +322,17 @@ declare namespace Realm {
...
-        static facebook(token: string): Credentials;
-        static apple(token: string): Credentials;
-        static gooogle(token: string): Credentials;
-        static anonymous(): Credentials;
-        static userAPIKey(key: string): Credentials;
-        static serverAPIKey(key: string): Credentials;
-        static custom(token: string): Credentials;
-        static function(token: string): Promise<Credentials>;
-    }
+    // class Credentials {
```
Comment (bmunkholm) [inline types/index.d.ts:325]: Why is this commented out? How about adding a comment or TODO?

File: types/index.d.ts [hunk 2]
```diff
@@ -403,13 +403,14 @@ declare namespace Realm {
...
     }
-    class App {
-        logIn(credentials: Credentials): Promise<User>;
-        allUsers(): UserMap;
-        currentUser(): User | null;
-        switchUser(user: User): void;
-        removeUser(user: User): Promise<User>;
-    }
+
+    // class App {
```
Comment (bmunkholm) [inline types/index.d.ts:407]: same

File: types/index.d.ts (hunk 1 above)
Comment (blagoev) [reply types/index.d.ts:325]: fixed

File: types/index.d.ts (hunk 2 above)
Comment (blagoev) [reply types/index.d.ts:407]: fixed

**Comment by blagoev** [comment]
removed dead tgz file in v10

**Pull request merged**